### ISSUE TITLE
Helm Release supports revision diff, rollback, upgrade

### ISSUE THREAD

**Issue by umialpha** [kubesphere/kubesphere#5563]
**What's it about?**

When using helm, we can easily check its revision, diff or rollback by cmd:
```
helm history
helm get values --revision
helm rollback
```
However, it is not supported by kubesphere.

In our private kubesphere branch, we have implemented this.

if community has interest in this feature, We can offer some help

![20230302-163853](https://user-images.githubusercontent.com/13465150/222375749-f48dc95f-9fd2-4067-aea5-ed247cd0f60e.png)

**What's the reason why we need it?**

**Area Suggestion**

/kind feature-request

/area apiserver
/area app-management

**Comment by umialpha** [comment]
@chilianyi @VioZhang , i mentioned in #5431 

what do you think this feature

**Comment by chilianyi** [comment]
Yes, after upgrade helm release, there are no history and rollback.

We are interested in your feature，could you submit a pr about this？

**Comment by umialpha** [comment]
sure. i will open a pr ASAP
